wvdecrypter/wvdecrypter.cpp

```diff
@@ -529,6 +535,8 @@ void WV_CencSingleSampleDecrypter::GetCapabilities(const uint8_t* key, uint32_t
   {
     AP4_UI32 poolid(AddPool());
     fragment_pool_[poolid].key_ = key ? key : reinterpret_cast<const uint8_t*>(keys_.front().keyid.data());
+    fragment_pool_[poolid].m_cryptoInfo = CryptoInfo();
```

Comment:
AddPool construct FINFO
so m_cryptoInfo is already constructed so add `= CryptoInfo()`
its not needed